pkgs/apps/gabedit/default.nix

```
@@ -0,0 +1,67 @@
+{ lib, stdenv, fetchurl, gtk2, pkg-config, xorg, libGL, libGLU, gnome2, pango }:
+
+stdenv.mkDerivation rec {
+  pname = "Gabedit";
+  version = "2.5.1";
+
+  src = with lib.versions; fetchurl {
```

```suggestion
  src = with lib.versions; fetchurl 
  let
    urlVersion = "${major version}${minor version}${patch version}"
  in {
```
to shorten the line below a bit?